docs/examples/tutorial/clibraries/queue3.py

```diff
@@ -0,0 +1,67 @@
+from cython.cimports import cqueue
+
+@cython.cclass
+class Queue:
+    """A queue class for C integer values.
+
+    >>> q = Queue()
+    >>> q.append(5)
+    >>> q.peek()
+    5
+    >>> q.pop()
+    5
+    """
+    _c_queue = cython.declare(cython.pointer(cqueue.Queue))
+    def __cinit__(self):
+        self._c_queue = cqueue.queue_new()
+        if self._c_queue is cython.NULL:
+            raise MemoryError()
+
+    def __dealloc__(self):
+        if self._c_queue is not cython.NULL:
+            cqueue.queue_free(self._c_queue)
+
+    @cython.ccall
+    def append(self, value: cython.int):
+        if not cqueue.queue_push_tail(self._c_queue,
+                                      cython.cast(cython.p_void, cython.cast(cython.Py_ssize_t, value))):
```

Comment:
Long lines are ok in editors, but not so great in documentation.
```suggestion
        if not cqueue.queue_push_tail(self._c_queue,
                cython.cast(cython.p_void, cython.cast(cython.Py_ssize_t, value))):
```